File: k9amqp.go
package k9amqp

import (
	"go.k6.io/k6/js/modules"
)

func init() {
	modules.Register("k6/x/k9amqp", new(K9amqp))
}

type K9amqp struct{}

func (k9amqp *K9amqp) Test() string {
	println("K9 AMQP")
	return "test"
}

File: k9amqp.js
import http from 'k6/http';
import { sleep } from 'k6';
import { k9amqp } from 'k6/x/k9amqp';

export const options = {
  vus: 10,
  duration: '30s',
}

export default function() {
  console.log(k9amqp);
  //console.log(k9amqp.test());
  sleep(1);
}
